---
layout: page
---

# Eli N. Weinstein

<img src="/images/Eli_Weinstein_square.jpg" alt="drawing" width="100" align="left" hspace="10">


I'm a 6th year PhD student in the Program in Biophysics at Harvard, advised by Debora Marks ([link](https://marks.hms.harvard.edu/index.html)) and Jeff Miller ([link](https://jwmi.github.io/)). I develop statistical tools for modern molecular biology, working in the fields of Bayesian statistics, probabilistic machine learning, biophysics and genomics.

My primary theoretical interests are in the statistical foundations of biological sequence analysis, from the perspective of Bayesian methodology. My primary applied interests are in understanding and predicting complex biological sequence dynamics, especially over very short timescales (programmed mutagenesis systems) and very long timescales (the whole of evolution). Other research themes include model misspecification, approximate Bayesian inference, Bayesian nonparametrics, sequence library design, and pathogen forecasting.

I received my A.B. from Harvard in 2016, concentrating in Chemistry and Physics. I worked with Adam Cohen, developing analysis methods for high-throughput single-cell all-optical electrophysiology data. My graduate research is supported by a Hertz Foundation Fellowship ([link](https://www.hertzfoundation.org/)).

Email: eweinstein [at] g.harvard.edu

### Selected publications and preprints

**Eli N. Weinstein**, Jeffrey W. Miller. *Bayesian data selection.* In submission. [preprint](https://arxiv.org/abs/2109.02712). [code](https://github.com/EWeinstein/data-selection).

**Eli N. Weinstein\***, Alan N. Amin\*, Jonathan Frazer, Debora S. Marks (\*Equal contribution). *Non-identifiability and the blessings of misspecification in models of molecular fitness and phylogeny.* In submission. [preprint](https://www.biorxiv.org/content/10.1101/2022.01.29.478324v1).

**Eli N. Weinstein**, Alan N. Amin, Will Grathwohl, Daniel Kassler, Jean Disset, Debora S. Marks. *Optimal design of stochastic DNA synthesis protocols based on generative sequence models.* Accepted at AISTATS (2022). [preprint](https://www.biorxiv.org/content/10.1101/2021.10.28.466307v1). [code](https://github.com/debbiemarkslab/variational-synthesis).

Alan N. Amin\*, **Eli N. Weinstein\***, Debora S. Marks (\*Equal contribution). *A generative nonparametric Bayesian model for whole genomes*. Advances in Neural Information Processing Systems (NeurIPS). 2021. [paper](https://proceedings.neurips.cc/paper/2021/hash/e9dcb63ca828d0e00cd05b445099ed2e-Abstract.html). [code](https://github.com/debbiemarkslab/BEAR). [talk](https://www.youtube.com/watch?v=bR8Ct75w3YE&t=2737s).

**Eli N. Weinstein**, Debora S. Marks. *A structured observation distribution for generative biological sequence prediction and forecasting.* Proceedings of the 38th International Conference on Machine Learning (ICML). 2021. [paper](http://proceedings.mlr.press/v139/weinstein21a.html). [Pyro code](https://docs.pyro.ai/en/dev/contrib.mue.html). [Edward2 code](https://github.com/debbiemarkslab/MuE). [talk](https://www.youtube.com/watch?v=bR8Ct75w3YE&t=2737s).

### Publications
My google scholar page is [here](https://scholar.google.com/citations?user=Tkv7cWAAAAAJ&hl=en).
